examples/package.json

```diff
@@ -21,38 +21,38 @@
   ],
   "devDependencies": {
     "@react-three/cannon": "*",
-    "@react-three/drei": "^8.11.1",
+    "@react-three/drei": "^8.19.2",
```

Comment:
Can we do this without bumping all of the dependencies at the same time?

Reply:
Sure, it wasn't necessary per se, just did it to see if anything would break